packages/scene-composer/src/components/modals/DeleteNodeModal.tsx

```diff
@@ -0,0 +1,117 @@
+import { isNumber } from 'lodash';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { Spinner } from '@cloudscape-design/components';
+
+import { sceneComposerIdContext } from '../../common/sceneComposerIdContext';
+import { useSceneDocument, useStore } from '../../store';
+import { getGlobalSettings } from '../../common/GlobalSettings';
+import { DEFAULT_PARENT_RELATIONSHIP_NAME, MAX_QUERY_HOP } from '../../common/entityModelConstants';
+
+import DeleteConfirmationModal from './DeleteConfirmationModal';
+
+const DeleteNodeModal: React.FC = () => {
```

Comment:
No need to use `React.FC `

Reply:
there will be lint warning for function without return type